src/components/CommentsList.jsx

```diff
@@ -1,12 +1,31 @@
 import CommentCard from "./CommentCard";
 
 export default function CommentsList({ comments }) {
-  return (
-    <section className="comment-list">
-      {comments.map((comment) => {
-        return <CommentCard key={comment.comment_id} comment={comment} />;
-      })}
-      <a href="#header">Back to top</a>
-    </section>
-  );
+  if (!comments.length) {
```

Comment:
an empty array would still have a length - think about how you check if an array.length is 0 :)